xbmc/windowing/XBMC_events.h

```diff
@@ -78,6 +80,10 @@ typedef struct XBMC_MoveEvent {
 	int y;		/* New y position */
 } XBMC_MoveEvent;
 
+struct XBMC_ModeChangeEvent {
```

Comment:
sure it makes sense. the fact that someting is wrong is no justification to continue with doing it wrong.